src/app/common/s/service/traveler-data.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TravelerDataService {
```

Comment:
I think that we should all coordinate on these services. They are so similar I don't see why there should be three of them.

Reply:
I'll update that in next pr